lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -736,20 +736,20 @@ ProxyRenderDelegate::ProxyShapeData::ProxyShapeData(const MayaUsdProxyShapeBase*
 {
     assert(_proxyShape);
 }
-const MayaUsdProxyShapeBase* ProxyRenderDelegate::ProxyShapeData::ProxyShape() const { return _proxyShape; }
-const MDagPath& ProxyRenderDelegate::ProxyShapeData::ProxyDagPath() const { return _proxyDagPath; }
-UsdStageRefPtr ProxyRenderDelegate::ProxyShapeData::UsdStage() const { return _usdStage; }
-void ProxyRenderDelegate::ProxyShapeData::UpdateUsdStage() { _usdStage = _proxyShape->getUsdStage(); }
-bool ProxyRenderDelegate::ProxyShapeData::IsUsdStageUpToDate() const {
+inline const MayaUsdProxyShapeBase* ProxyRenderDelegate::ProxyShapeData::ProxyShape() const { return _proxyShape; }
```

Comment:
I assume these inline functions won't be called from other cpp files?

Info from https://isocpp.org/wiki/faq/inline-functions#overview-inline-fns

> The reason you (almost always) put the definition (the {...} part) of an inline function in a header file is to avoid “unresolved external” errors from the linker. That error will occur if you put the inline function’s definition in a .cpp file and if that function is called from some other .cpp file.

Reply:
Yes no other cpp files need them. If we wanted to use ProxyShapeData in some other class we'd have to move them to the header but we're not doing that right now, and this avoids head bloat of include proxyShapeBash.h in proxyRenderDelegate.h.